lib/content-services/src/lib/node-comments/services/content-comment-list.service.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CommentListService, PeopleContentService } from '@alfresco/adf-core';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ContentCommentListService implements CommentListService {
+
+  constructor(private peopleContentService: PeopleContentService) { }
+
+  getUserImage(user: string): string {
+    return this.peopleContentService.getUserProfileImage(user);
+  }
+}
```

Comment:
I wonder if we need this service at all. All it does is calling another service. Can't we remove it and just use the peopleContentService directly wherever needed?

Reply:
I went with that approach as to maintain the same structure across content-services and process-services. No matter what you want the user image to be in the comment-list component, it will always be named `getUserImage()`